CHANGELOG.md

```diff
@@ -3,6 +3,20 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 
+## [3.0.2-rc0] - 2020-10-30
+### Changed or Fixed
+- FAPI: Fix setting of the system flag of NV objects
+  This will let NV object metadata be created system-wide always instead of
+  locally in the user. Existing metadata will remain in the user directory.
+  It can be moved to the corresponding systemstore manually if needed.
+- FAPI: Set the written flag of NV objects in FAPI PolicyNV commands
+- FAPI: Fix deleting of policy files.
+- FAPI: Fix wrong file loading during object search.
+- Fapi: Fix memory leak
+- Fapi: Fix potential NULL-Dereference
```

Comment:
Coverity is showing another potential NULL-Dereference here (just FYI): https://scan8.coverity.com/reports.htm#v22486/p13007/fileInstanceId=37005548&defectInstanceId=11222604&mergedDefectId=312963

![image](https://user-images.githubusercontent.com/5950433/97891939-445d0480-1ce4-11eb-821d-0a5808204daf.png)

Reply:
@JuergenReppSIT Could you have a look at this ?